File: 2744. Find Maximum Number of String Pairs/Solution.py
class Solution:
    def maximumNumberOfStringPairs(self, words: List[str]) -> int:
        count, n = 0, len(words)
        for i in range(n):
            rev = words[i][::-1]
            for j in range(i + 1, n):
                if rev == words[j]:
                    count += 1
                    break
        return count

File: 2744. Find Maximum Number of String Pairs/Solution.java
class Solution {
    public int maximumNumberOfStringPairs(String[] words) {
        int count = 0, n = words.length;
        for (int i = 0; i < n; i++) {
            StringBuilder temp = new StringBuilder(words[i]);
            String rev = temp.reverse().toString();
            for (int j = i + 1; j < n; j++) {
                if (rev.equals(words[j])) {
                    count++;
                    break;
                }
            }
        }
        return count;
    }
}
